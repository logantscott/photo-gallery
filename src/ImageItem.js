import React, { Component } from 'react';

export default class ImageItem extends Component {

    render() {
        return (
            <div>

                This is my Image Item.

            </div>
        )
    }
}