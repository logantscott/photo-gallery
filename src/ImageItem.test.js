import React from "react";
import ImageItem from "./ImageItem.js";
import renderer from "react-test-renderer";

test('renders ImageItem.js correctly', () => {
  const tree = renderer
    .create(<ImageItem image={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});