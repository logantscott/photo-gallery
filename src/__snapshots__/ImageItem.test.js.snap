// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ImageItem.js correctly 1`] = `
<div
  className="grid-item"
>
  <img />
  <h3 />
  <p />
</div>
`;
